docstore/doc.go

```diff
@@ -51,17 +51,66 @@
 // information.
 //
 //
-// Documents
+// Representing Documents
+//
+// A document is a set of named fields, each with a value. Values can themselves
+// be documents or lists of values.
```

Comment:
also scalars?

Reply:
I added "scalars" and reworded the sentence, but I'm not totally happy with it. Suggestions appreciated.

Reply:
Done.